hints/cygwin.sh

```diff
@@ -62,13 +71,23 @@ case "$osvers" in
         d_inetpton='undef'
 esac
 
+case "$osvers" in
+    2.[0-4].*|1.*)
+        # newlib finitel is buggy before cygwin-2.5.0
+        d_finitel='undef'
+        ;;
+esac
```

Comment:
Cygwin is one of those projects that you have to have a very good reason to use a non-latest version. They don't even make them available for download. IMO this part should be skipped.

Reply:
That sounds reasonable